frontend/src/assets/css/reboot.scss

```diff
@@ -0,0 +1,684 @@
+// stylelint-disable at-rule-no-vendor-prefix, declaration-no-important, selector-no-qualifying-type, property-no-vendor-prefi
+@import "variables";
+@import "vendor/rfs.scss";
+
+// These variables are needed for reboot. We supply them from bootstrap's
+// variable files
+$font-size-root: null !default;
+$font-family-base: $font-family-sans-serif !default;
+$font-family-code: $font-family-monospace !default;
+$body-bg: $white !default;
+$body-color: $gray-900 !default;
+$body-text-align: null !default;
+$text-muted: $gray-600 !default;
+$border-width: 1px !default;
+$font-weight-normal: 400 !default;
+$font-weight-bold: 700 !default;
+$font-weight-bolder: bolder !default;
+$font-weight-base: $font-weight-normal !default;
+$hr-margin-y: $spacer !default;
+$hr-color: inherit !default;
+$hr-height: $border-width !default;
+$hr-opacity: 0.25 !default;
+$headings-margin-bottom: $spacer / 2 !default;
+$headings-font-family: null !default;
+$headings-font-style: null !default;
+$headings-font-weight: 500 !default;
+$headings-line-height: 1.2 !default;
+$headings-color: null !default;
+$h1-font-size: $font-size-base * 2.5 !default;
+$h2-font-size: $font-size-base * 2 !default;
+$h3-font-size: $font-size-base * 1.75 !default;
+$h4-font-size: $font-size-base * 1.5 !default;
+$h5-font-size: $font-size-base * 1.25 !default;
+$h6-font-size: $font-size-base !default;
+$paragraph-margin-bottom: 1rem !default;
+$dt-font-weight: $font-weight-bold !default;
+$small-font-size: 0.875em !default;
+$mark-padding: 0.2em !default;
+$mark-bg: #fcf8e3 !default;
+$sub-sup-font-size: 0.75em !default;
+$link-color: $primary !default;
+$link-decoration: underline !default;
+$link-hover-color: darken($link-color, 15%) !default;
+$link-hover-decoration: null !default;
+$input-btn-padding-y: 0.375rem !default;
+$code-font-size: $small-font-size !default;
+$kbd-padding-y: 0.2rem !default;
+$kbd-padding-x: 0.4rem !default;
+$kbd-font-size: $code-font-size !default;
+$enable-rounded: true !default;
+$nested-kbd-font-weight: $font-weight-bold !default;
+$table-cell-padding-y: 0.5rem !default;
+$table-caption-color: $text-muted !default;
+$table-th-font-weight: null !default;
+$enable-button-pointers: true !default;
+$legend-margin-bottom: 0.5rem !default;
+$legend-font-size: 1.5rem !default;
+$legend-font-weight: null !default;
+$grid-breakpoints: (
+  xs: 0,
+  sm: 576px,
+  md: 768px,
+  lg: 992px,
+  xl: 1200px,
+  xxl: 1400px,
+) !default;
+
+@mixin border-radius($radius: $border-radius, $fallback-border-radius: false) {
+  @if $enable-rounded {
+    border-radius: valid-radius($radius);
```

Comment:
I'm not sure this mixin works? Not sure where valid-radius is from. It compiles fine in our codebase but sassmeister does not like. Also I don't see the border radius applied here

![image](https://user-images.githubusercontent.com/24946400/94856150-8d781b00-03fd-11eb-87c9-008d9668fe91.png)